client/app/components/template-explorer/template-explorer.html

```diff
@@ -0,0 +1,60 @@
+<div class="breadcrumb-bar">
+  <span>
+      <ol class="breadcrumb">
+        <li class="active">{{'Templates'|translate}}</li>
+      </ol>
+  </span>
+</div>
+<div pf-toolbar class="section-toolbar section-toolbar-right-actions" config="vm.toolbarConfig">
+  <actions>
+    <custom-dropdown
+      class="custom-dropdown pull-left"
+      config="item"
+      items="vm.selectedItemsList"
+      items-count="vm.selectedItemsList.length"
+      ng-repeat="item in vm.actionConfig"
+      on-update="vm.listActionDisable($config, $changes)"
+      menu-right="true">
+    </custom-dropdown>
+  </actions>
+</div>
+<div class="list-view-container section-container paged-container service-explorer-list">
+  <loading status="vm.loading"></loading>
+  <div ng-if="!vm.loading"
+       pf-list-view id="templatesList"
+       config="vm.listConfig"
+       items="vm.templatesList"
+       custom-scope="vm"
+       menu-actions="vm.menuActions">
+    <div class="row">
+      <div class="col-lg-4 col-md-4 col-sm-4 col-xs-8">
```

Comment:
for sm and xs column totals don't add up to 12, means we'll either overflow or have whitespace